src/get-started/focus-states/index.md.njk

```diff
@@ -0,0 +1,83 @@
+
+Other components and elements that look like links use the link focus state style. For example, the controls on the [accordion](/components/accordion/) and [details](/components/details/) components.
+
+![A focused details component. In the examples, the expandable text reads "Help with nationality" and beneath is an explanation of why the user is being asked for this information.](details-focus.png)
+
+## Form input focus state style
+
+When form inputs are focused, they have a yellow outline and a thick black border. If the element already has a border, the border gets thicker.
+
+![A text input labelled "What is your name?". The example shows the text input both unfocused and focused.](text-input-focus.png)
+
+[Radios](/components/radios/) and [checkboxes](/components/checkboxes/) use the same style.
+
+![Yes and no radio options to answer the question "Have you changed your name?". In this example, the "No" option is focused.](radios-focus.png)
+
+## Making focus states accessible for extended and modified components
+
+If you've [extended or modified components in the GOV.UK Design System](/get-started/extending-and-modifying-components/), you can use GOV.UK styles to make the focus states of these components accessible.
+
+How you make focus states accessible depends on if the component is:
+
+- focusable text without a background colour or border
+- another focusable element with a background colour or border
+
+### Make focusable text accessible
+
+If you use Sass, you should include the `govuk-focused-text` mixin in your component's `:focus` selector if that component is focusable text. For example, the component is a link in body text, or the details component:
+
+```
+.app-component:focus {
+@include govuk-focused-text;
+}
+```
+### Make other focusable elements accessible
+
+If you use Sass, you can use 3 GOV.UK Frontend variables if your component has a background colour or border. For example a text input or checkbox.
+
+The 3 Sass variables are:
+
+- `govuk-focus-colour` - yellow background
+- `govuk-focus-text-colour` - black text 
+- `govuk-focus-width` - for consistent width
+
+Use these variables in your components instead of numeric values for the background, text and widths.
+
+To test the focus states in your components are still accessible, you can:
+
+- [turn on high contrast in Windows](https://support.microsoft.com/en-gb/help/4026951/windows-10-turn-high-contrast-mode-on-or-off) 
+- install the [High Contrast extension](https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph?hl=en-GB) in Chrome
+- change colours in Firefox
+
+## If you do not use Sass
+
+To make a component's focus state accessible without using Sass, you can:
+
+- see how the `govuk-focused-text` mixin works from the [GOV.UK Frontend source code](https://github.com/alphagov/govuk-frontend/blob/268b1e1b962d4bd293c5681868a2119f8b4ece60/src/govuk/helpers/_focused.scss#L12-L33) 
```

Comment:
We can update this link to the newer version of these styles: https://github.com/alphagov/govuk-frontend/blob/25a4333b239e1c3b8a136e526981fe29172a2852/src/govuk/helpers/_focused.scss#L12-L28